awesome.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+RESULT_CODE_RESTART=8
+
+result=$RESULT_CODE_RESTART
+while [ $result == $RESULT_CODE_RESTART ]; do
+    awesome
+    result=$?
+done
```

Comment:
Could do `while true; do … ` with a `break` to check the result only after running awesome.